packages/image_picker/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
-## 0.6.2+2
+## 0.6.2+3
```

Comment:
add an extra line for consistency.

Reply:
Thanks! I'll follow-up in a separate PR, I want to avoid running all presubmits for all plugins just for this 😄